app/components/modal-component.coffee

```diff
@@ -39,21 +47,21 @@ Ember.Component.extend Ember.Widgets.StyleBindingsMixin,
     templateName: 'modal-footer'
 
   _headerViewClass: Ember.computed ->
-    headerViewClass = @get 'headerViewClass'
+    headerViewClass = Ember.get @, 'headerViewClass'
```

Comment:
use `this` instead of `@` when used on its own. Applies to a few places below as well.